pkgs/applications/video/clapper/default.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, stdenv, fetchFromGitHub,
+  glib,
+  gobject-introspection,
+  python3,
+  cmake,
+  pkgconfig,
+  ninja,
+  libGL,
+  wayland,
+  wayland-protocols,
+  desktop-file-utils,
+  makeWrapper,
+  shared-mime-info,
+  meson, gjs, gtk4, gst_all_1}:
+
+stdenv.mkDerivation rec {
+  pname = "clapper";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner  = "Rafostar";
+    repo   = pname;
+    rev    = version;
+    sha256 = "1761rp13a3kw6746y89bfdjyc2p156kq58y3rk2zygzryl6fpjf8";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    desktop-file-utils # for update-desktop-database
+    glib
+    gobject-introspection
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+    meson
+    ninja
+    makeWrapper
+    pkgconfig
+  ];
+  buildInputs = [
+    gjs
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
```

Comment:
I don't think we should be depending on this package, and instead should rely on gstreamer inspection.

Reply:
Totem includes ugly in its buildInputs. I'll keep this as is.